lib/displays/programs/contact.php

```diff
@@ -0,0 +1,8 @@
+<?php namespace WSUWP\CAHNRSWSUWP_Plugin_Extension_Core;
+
+?><div class="program-contact">
+<?php if ( ! empty( $program_contact['icon'] ) ) : ?><img src="<?php echo esc_url( ecore_get_plugin_url( 'lib/images/' . $program_contact['icon'] . '.png' ) ); ?>" height="75" width="75" class="extension-program-icon" /><?php endif; ?>
```

Comment:
`$program_contact['icon']` should be escaped here as it's pulled from meta in the `get_program_contact_array()` method and can't be trusted.